addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -237,9 +237,17 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                 else:
                     to_return[invoice] = {'attachment': invoice.l10n_it_edi_attachment_id, 'success': True}
             elif state == 'notificaScarto':
-                errors = [element.find('Descrizione').text for element in response_tree.xpath('//Errore')]
-                to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors), 'blocking_level': 'error'}
-                invoice.l10n_it_edi_transaction = False
+                elements = response_tree.xpath('//Errore')
+                error_codes = [element.find('Codice').text for element in elements]
+                errors = [element.find('Descrizione').text for element in elements]
+                # Duplicated invoice
+                if '00404' in error_codes:
+                    idx = error_codes.index('00404')
+                    invoice.message_post(body=errors[idx])
```

Comment:
@lordkrandel We should put a message like: "This invoice number had already been submitted so it is considered done.  But this could also be because of an initial bad configuration on your side, so be careful. The error given by the system is this one:"